backend/src/index.ts

```diff
@@ -0,0 +1,50 @@
+import dotenv from "dotenv";
+import express, { Request, Response } from "express";
+import cors from "cors";
+import helmet from "helmet";
+import http from "http";
+import { Server as IOServer, Socket } from "socket.io";
+
+dotenv.config();
+
+// Setup Environment Variables
+if (!process.env.PORT) {
+  console.error('ERROR: Environment variable "PORT" not set');
+  process.exit(1);
+}
+const PORT: number = parseInt(process.env.PORT as string, 10);
+
+// Setup Express
+const app = express();
+
+app.use(helmet());
```

Comment:
What's this dependency do?

Reply:
> Helmet helps you secure your Express apps by setting various HTTP headers. It's not a silver bullet, but it can help!

[Helmet](https://github.com/helmetjs/helmet) is a nice wee middleware for Express that sets HTTP headers to improve security & is considered [best practice](http://expressjs.com/en/advanced/best-practice-security.html)